Makefile

```diff
@@ -76,11 +76,12 @@ goword:tools/bin/goword
 gosec:tools/bin/gosec
 	tools/bin/gosec $$($(PACKAGE_DIRECTORIES))
 
-check-static:tools/bin/gometalinter
-	@ # vet and fmt have problems with vendor when ran through metalinter
-	tools/bin/gometalinter --disable-all --deadline 120s \
+check-static:tools/bin/gometalinter tools/bin/misspell tools/bin/ineffassign
+	@ # TODO: enable megacheck.
+	@ # TODO: gometalinter has been DEPRECATED.
+	@ # https://github.com/alecthomas/gometalinter/issues/590
```

Comment:
Do we need to change `gometalinter` to [golangci-lint](https://github.com/golangci/golangci-lint)?

Reply:
Is it easy to switch to golangci-lint in this PR? If so, please do it.

Reply:
I test it.
And I think it is not stable enough. (I met this problem https://github.com/golangci/golangci-lint/issues/401)
I think we would better switch to golangci-lint when our repo upgrades to go1.12.

Reply:
OK.